tests/Integration/Functions/rocketGetCompressedAssets.php

```diff
@@ -0,0 +1,79 @@
+<?php
+namespace WP_Rocket\Tests\Integration\Functions;
+
+use WP_Rocket\Tests\Integration\FilesystemTestCase;
+use Brain\Monkey\Functions;
+
+/**
+ * @covers rocket_get_compressed_assets_rules()
+ * @group Functions
+ * @group Htaccess
+ * @group AdminOnly
+ */
+class Test_RocketGetCompressedAssetsRules extends FilesystemTestCase {
+	protected $structure = [
+		'.htaccess' => '',
+	];
+
+	public function setUp() {
+		parent::setUp();
+
+		global $is_apache;
+
+		$is_apache = true;
+
+		Functions\when( 'rocket_valid_key' )->justReturn( true );
```

Comment:
@Tabrisrp This function is redefined in the bootstrap file and set to `true`. Did it not work for you?

Reply:
During debugging, I found out that it was returning true on the first call, and other calls were still returning false. So I had to force the value in the test here

Reply:
Good to know. Thank you.